src/erlfmt.erl

```diff
@@ -91,6 +107,18 @@ format_file(FileName, Options) ->
 -spec format_string(string(), config()) ->
     {ok, string(), [error_info()]} | {skip, string()} | {error, error_info()}.
 format_string(String, Options) ->
+    Range = proplists:get_value(range, Options),
+    case Range of
+        undefined ->
+            % Whole file (default).
+            format_string_full(String, Options);
```

Comment:
Nice

Reply:
I really like how well these functions are named and how easily this reads in both `format_string` and `format_file` functions.